tmdl_tool.cs

```diff
@@ -338,10 +316,10 @@ private static void Deploy_TMDL(Server server, string datasetName, string tmdlfo
 
                 // Check if we have an access token in Environment variable
                 var tokenString = Environment.GetEnvironmentVariable("tmdl_accesstoken");

```

Comment:
Is the environment variable check still needed if we can use settings file or cli args?

Reply:
and if it is still needed then there should be one for tmdl_appsecret too